docs/computing/webinterface/jupyter.md

```diff
@@ -42,8 +46,27 @@ Then enter the full path to the python interpreter of your Tykky installation. S
 
 ### Virtual environment
 
-You can create a virtual environment by enabling the virtual environment in the app form and providing a path under `/scratch` or `/projappl` to where you want to store the environment.  
-To launch a created virtual environment later you need to select the same Python module and virtual environment path as when creating the environment.
-The virtual environments are currently not completely isolated as they use packages from the loaded modules.
+To create a virtual environment, connect to Puhti from your terminal and then navigate to either `/scratch` or `/projappl` folder and then create the environment with a command:   
+`python -m venv --system-site-packages <venv>`      
+Make sure to load the module you are planning to use before creating the virtual environment.
+
+You also can create a virtual environment by enabling the virtual environment in the app form and providing a path under `/scratch` or `/projappl` to where you want to store the environment.
```

Comment:
We might want to these instructions before the instructions for how to manually create the environment since this this does some extra things that the user might also have to do.
In particular, installing jupyterlab and installing a Jupyter kernel might need to be done if the module doesn't provide them (I think most do though).
Previously we had also had a lot of extra patching that needed to be done for virtual environments with containerized python installation, but I think that might not be needed at all anymore. Either way, the safer way for most users would be to create the environment through the form.

Reply:
Fixed in new commit